recipes/hxrsnd/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "hxrsnd" %}
+{% set version = "0.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/pcdshub/hxrsnd/archive/v{{ version }}.tar.gz
+  sha256: fcc6256436225b4df52ffd1c32ec7c795a5d65447808a97e1549196bc87e3415
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - bluesky
+    - coloredlogs
+    - lmfit
+    - numpy
+    - ophyd >=1.4.0
+    - pandas
+    - pcdsdevices
+    - pswalker
+    - pyyaml
+    - scipy
+    - simplejson
+    - super_state_machine
+
+test:
+  imports:
+    - hxrsnd
+
+about:
+  home: https://github.com/pcdshub/hxrsnd
+  doc_url: https://pcdshub.github.io/hxrsnd
+  dev_url: https://github.com/pcdshub/hxrsnd
+  license: SLAC Open
+  license_family: Other
+  license_file: LICENSE
```

Comment:
LICENSE.md but not at the MANIFEST.in maybe add the LICENSE here?

Reply:
Addressed in fba14e328